devops/deploy/spin-down-script.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
```

Comment:
Just noticed the shebang. In general it's better to use `#!/usr/bin/env <program>`, especially for scripts that will be used widely. Though it's not _always_ the case.

Some good discussion in this [SO thread](https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash)